code/modules/power/apc.dm

```diff
@@ -1270,10 +1249,36 @@ var/zapLimiter = 0
 
 	src.updateDialog()
 
+// set channels depending on how much charge we have left
+/obj/machinery/power/apc/proc/check_channle_thresholds()
```

Comment:
Spelling - I think this should be check_channel_thresholds()

Reply:
Aye! thanks for catching that!